app/javascript/packs/articlePage.jsx

```diff
@@ -63,25 +63,19 @@ if (shareDropdownButton.dataset.initialized !== 'true') {
 
 // Initialize the copy to clipboard functionality
 function showAnnouncer() {
-  const { activeElement } = document;
-  const input =
-    activeElement.localName === 'clipboard-copy'
-      ? activeElement.querySelector('input')
-      : document.getElementById('article-copy-link-input');
-  input.focus();
-  input.setSelectionRange(0, input.value.length);
-
   document.getElementById('article-copy-link-announcer').hidden = false;
 }
 
 function copyArticleLink() {
-  const inputValue = document.getElementById('article-copy-link-input').value;
+  const inputValue = document
```

Comment:
suggestion (not blocking) - it's not a strong opinion at all so feel free to ignore, but maybe since we're no longer retrieving this value from an input we should rename it - `postUrlValue`?